src/main/java/cz/smarteon/loxone/user/NfcTagSerializer.java

```diff
@@ -0,0 +1,24 @@
+package cz.smarteon.loxone.user;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import java.io.IOException;
+import java.util.stream.Stream;
+
+public class NfcTagSerializer extends StdSerializer<NfcTag> {
```

Comment:
could be package private?

Reply:
or may be make it inner class of where it's used - as we do with other single-place-used serializers?